2019/06/08 22:37:34 [error] 6#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: symfony.local, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost"
2019/06/08 22:37:34 [error] 6#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: symfony.local, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost", referrer: "http://localhost/"
2019/06/10 03:34:08 [error] 8#0: *1787 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.18.0.1, server: symfony.local, request: "GET /_profiler/872149?panel=exception HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost", referrer: "http://localhost/"
2019/06/10 03:35:01 [error] 8#0: *1787 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 172.18.0.1, server: symfony.local, request: "GET /_profiler/872149?panel=exception HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost", referrer: "http://localhost/"
2019/06/10 03:35:53 [error] 7#0: *1791 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 172.18.0.1, server: symfony.local, request: "GET /_profiler/872149?panel=exception HTTP/1.1", upstream: "fastcgi://172.18.0.3:9000", host: "localhost", referrer: "http://localhost/"
